lib/gc/Transforms/IterativeTilingAndFusion.cpp

```diff
@@ -228,19 +228,37 @@ NonContractionOpFilter(RewriterBase &rewriter,
   return failure(isa<mlir::linalg::ContractionOpInterface>(defOrUse.ownerOp));
 }
 
+/// If fusing multiple consumers is allowed, there may exist following cases:
+///
+/// ```
+/// %1:2 = scf.for() {
+///     %2 = tiled_op1
+///     %3 = tiled_op2
+///     %4 = insert_slice %2
+///     %5 = insert_slice %3
+///     yield %4, %5
+/// }
+/// op3 ins(%1#1, %1#2)
+/// ```
+///
+/// Where we need to ensure their `tileOffset` and `tileSize` are matched well.
 static LogicalResult
-SingleCandidateInBlockFilter(RewriterBase &rewriter,
-                             OffsetSizeAndStrideOpInterface candidate,
-                             CandidateDefOrUse defOrUse) {
+SingleCandidateElseTileMatchedFilter(RewriterBase &rewriter,
```

Comment:
This filter is doing something like:
```
if(singleCandidate){
   return true;
} else {
  // check tile is matched.
}
```
So, I think `else` is more close to `otherwise` meaning.

Reply:
Maybe `SingleCandidateOrMultiCandidateTileMatchedFilter`? The current func name looks very confusing.

Reply:
Changed to `tilingSizesIfMatchedFilter`.